### PR TITLE
Fix #48. Add UPM v0.7.0 to edison base images. 

### LINKED ISSUE #48: Install UPM module on edison images
We should install UPM, which is a high level repository for sensors that use mraa, on edison images. UPM main page: http://iotdk.intel.com/docs/master/upm/

### PR DESCRIPTION
**Pull request by nghiant2710**
UPM provides software drivers for a wide variety of commonly used sensors
and actuators.

### CODE REVIEW

File: device-base/Dockerfile.i386.edison.tpl [hunk 1]
```diff
@@ -16,28 +16,48 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
...
 		libpcre3-dev \
 		python-dev \
 		swig \
+		pkg-config \
+		curl \
 	' \
 	&& apt-get update && apt-get install -y $buildDeps --no-install-recommends && rm -rf /var/lib/apt/lists/* \
+	&& mkdir /cmake \ 
+	&& curl -SL https://cmake.org/files/v3.5/cmake-3.5.2.tar.gz -o cmake.tar.gz \
+	&& echo "92d8410d3d981bb881dfff2aed466da55a58d34c7390d50449aa59b32bb5e62a  cmake.tar.gz" | sha256sum -c - \
```
Comment (petrosagg) [inline device-base/Dockerfile.i386.edison.tpl]: Why do we install cmake from source?

File: device-base/Dockerfile.i386.edison.tpl [hunk 2]
```diff
@@ -16,28 +16,48 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
...
+	&& tar -xzf cmake.tar.gz -C /cmake --strip-components=1 \
+	&& cd /cmake \
+	&& ./configure \
+	&& make -j $(nproc) \
+	&& make install \
+	&& cd / \
 	&& git clone https://github.com/intel-iot-devkit/mraa.git \
-	&& cd mraa \
+	&& git clone https://github.com/intel-iot-devkit/upm.git \
```
Comment (petrosagg) [inline device-base/Dockerfile.i386.edison.tpl]: Can move this line right above the `cd /upm`? It's looks out of order

File: device-base/Dockerfile.i386.edison.tpl (hunk 1 above)
Comment (nghiant2710) [inline device-base/Dockerfile.i386.edison.tpl]: cmake on wheezy is too old while UPM requires newer cmake version so we need to install from source @petrosagg

File: device-base/Dockerfile.i386.edison.tpl (hunk 2 above)
Comment (nghiant2710) [inline device-base/Dockerfile.i386.edison.tpl]: Updated!

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**